src/bindings/js/node/src/helper.cpp

```diff
@@ -183,6 +183,26 @@ Napi::Array cpp_to_js<ov::PartialShape, Napi::Array>(const Napi::CallbackInfo& i
     return arr;
 }
 
+ov::TensorVector parse_input_data(const Napi::Value& input) {
+    ov::TensorVector parsed_input;
+    if (input.IsArray()) {
+        auto inputs = input.As<Napi::Array>();
+        for (size_t i = 0; i < inputs.Length(); ++i) {
+            parsed_input.emplace_back(cast_to_tensor(static_cast<Napi::Value>(inputs[i])));
+        }
+    } else if (input.IsObject()) {
+        auto inputs = input.ToObject();
+        const auto& keys = inputs.GetPropertyNames();
+        for (size_t i = 0; i < keys.Length(); ++i) {
+            auto value = inputs.Get(static_cast<Napi::Value>(keys[i]).ToString().Utf8Value());
+            parsed_input.emplace_back(cast_to_tensor(static_cast<Napi::Value>(value)));
+        }
+    } else {
+        OPENVINO_THROW("parse_input_data(): wrong arg");
```

Comment:
Does it work properly? I mean, does this appear as exception in JS side and stop code processing?

Reply:
`parse_input_data()` is a helper method that won't be exposed. `OPENVINO_THROW` has to be caught and handled in the methods that are exposed to JS.